keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -407,6 +416,24 @@ marked UnschedulableAndUnresolvable.
 [volume restrictions plugin]: https://github.com/kubernetes/kubernetes/blob/v1.21.0/pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go#L29
 [node info cache]: https://github.com/kubernetes/kubernetes/blob/v1.21.0/pkg/scheduler/framework/types.go#L357
 
+##### Beta
+
+Support for pod preemption is enforced in beta.
+
+When a pod (A) is scheduled, if another pod (B) is found using the same PVC, the
+PVC uses ReadWriteOncePod, and pod (A) has higher priority than pod (B), then
+return Unschedulable (which will cause pod (B) to be preempted). If pod (A) has
+lower or equal priority compared with pod (B), return
+UnschedulableAndUnresolvable.
+
+In the PreFilter phase of the volume restrictions scheduler plugin, we will
```

Comment:
As discussed offline, it might be better to populate the cache during Filter instead, to avoid calculations for unnecessary nodes in big clusters.

However, I believe we can leave those details to the implementation PR review.